### PR TITLE
Thank you very much

### PR DESCRIPTION
**Pull request by ghost-code**
☕

### CODE REVIEW

**Comment by SlaunchaMan** [comment]
😢

**Comment by SlaunchaMan** [comment]
What if instead we just decrease the probability that it appears?

**Comment by ghost-code** [comment]
Is there a way for slackbot to use probability in responses?

File: scripts/thankyou.coffee [hunk 1]
```diff
@@ -1,19 +0,0 @@
...
-#
-# Commands:
-#
-# Author:
-#   Scumbag Joe
-
-module.exports = (robot) ->
-  robot.hear /\(thank you\)|(\bthank you\b)/i, (msg) ->
-    if msg.match[1] is 'thank you'
-      if Math.random() > .8
```
Comment (SlaunchaMan) [inline scripts/thankyou.coffee]: This is where the probability comes in. Looks like this is doing it 20% of the time?

Comment (ghost-code) [inline scripts/thankyou.coffee]: okay, incoming fix

**Comment by SlaunchaMan** [comment]
LGTM

**Pull request merged**